test/box_test.jl

```diff
@@ -0,0 +1,41 @@
+using HypothesisTests, Base.Test
+
+sim_data_h0=[0.297287984535462;0.382395967790608;-0.597634476728231;-0.0104452446373756;-0.839026854388764;0.311111338498334;2.29508782383731;-2.26708634880053;0.529965576166746;0.431421526422912;0.583708287568779;0.963271605038191;0.458790955053717;-0.522336757421508;0.408395838324752;-0.0504512299336653;-0.693653643803886;-1.77383325340806;0.120765964937439;-0.757633037720528;-1.72975618139069;0.795948622004616;0.670061981256062;0.550852377500761;-0.0633745924295606;1.33694278225092;-0.0731486333377393;-0.745464316640756;-1.22005512853465;-0.0531773375276925;-0.165136357802834;-2.11537024842731;-0.0667679865065892;1.22246184846806;0.567695459738444;0.763498928323387;0.378598879853350;-0.645597056636597;-0.664646244391031;-1.80302632421089;-1.28904760384974;-0.335270079518118;0.0704676013611580;0.341793767379823;1.73516566605438;1.29991596431019;0.206364081402617;-1.00885995553760;-0.850055670315398;1.12940943588433;-1.34853345649183;1.24029090042906;-0.0556470932073243;0.759390213945460;-0.0302668866467625;-0.361082667756094;-2.00730320253256;-0.568859127346638;-1.14962752439700;1.88062217463119;-1.47062454138738;-0.535210970165732;-0.963543959825501;-1.38510806767591;0.134300237561270;-0.616116666008819;-1.71998935631974;0.320769279987411;-1.44737289788151;-0.492271188069566;-0.413087383983183;1.00192851245595;1.17468132597511;-1.32697034506205;-1.59321486804407;-0.438709975758797;1.22579749799787;1.48909027806961;2.15943318710085;-0.909479018037503;1.14782030061250;-0.505072196522373;-0.267067035218130;1.49908239393073;0.797304160175390;-0.171640443307097;-0.469079609930096;0.217624021208600;0.359146407602350;0.320024715708058;0.259216328055686;0.459695985041628;0.209986360042143;1.52317539117726;1.25979460361220;-0.165283101114261;0.156194494887636;-0.814315294376699]
```

Comment:
I just noticed how incredibly long this line is. I'd probably break and align after every fourth or fifth semicolon (which could equivalently be commas, FYI--they construct the same thing). Typically we like to stick to the code style used in Base Julia, which sets a cap on line lengths at 92 characters.

A common style in this scenario is to do
```julia
sim_data_h0 = [
    0.297287984535462, 0.382395967790608, -0.597634476728231, -0.0104452446373756,
    -0.839026854388764, 0.311111338498334, 2.29508782383731, -2.26708634880053,
    ...
]
```